tests/tools/oecert/host/host.cpp

```diff
@@ -262,17 +268,121 @@ static oe_result_t _gen_report(
     return result;
 }
 
+static oe_result_t _generate_evidence(
+    oe_enclave_t* enclave,
+    const char* evidence_filename,
+    const char* endorsements_filename)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    size_t evidence_size = 0;
+    uint8_t evidence[65536];
+    size_t endorsements_size = 0;
+    uint8_t endorsements[65536];
+
+    OE_CHECK_MSG(
+        get_plugin_evidence(
+            enclave,
+            &result,
+            evidence,
+            sizeof(evidence),
+            &evidence_size,
+            endorsements,
+            sizeof(endorsements),
+            &endorsements_size),
+        "Failed to create evidence. Error: %s\n",
+        oe_result_str(result));
+
+    printf("get_plugin_evidence succeeded\n");
+    printf("evidence_size = %zu\n", evidence_size);
+
+    // Write evidence to file
+    {
+        FILE* output = NULL;
+#ifdef _WIN32
+        fopen_s(&output, evidence_filename, "wb");
+#else
+        output = fopen(evidence_filename, "wb");
+#endif
```

Comment:
This file already includes <openenclave/host.h> which has (in lines 35-48) mappings to allow code to just use the C11 _s APIs and not have to use ifdefs in all these files.  Consider adding fopen_s to that list there and then just using line 302 here without ifdefs, and same for 346 etc.   fopen_s has been part of C11 (i.e., standard C since 2011) per https://en.cppreference.com/w/c/io/fopen

E.g.,
#define fopen_s(pfp, name, mode) *(pfp) = fopen((name), (mode))

Reply:
Thanks for the pointer, I wasn't aware of those macros yet. All fixed now.